cmd/ethrex/initializers.rs

```diff
@@ -396,9 +396,10 @@ pub async fn init_l1(
 
     let store = match init_store(datadir, genesis).await {
         Ok(store) => store,
-        Err(StoreError::IncompatibleDBVersion) => {
+        Err(err @ StoreError::IncompatibleDBVersion { .. })
+        | Err(err @ StoreError::NotFoundDBVersion { .. }) => {
```

Comment:
We start at 1 so we can treat 0 and missing as equivalent, so I don't think the new error kind is necessary.